microprofile-lra/pom.xml

```diff
@@ -0,0 +1,255 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wildfly.quickstarts</groupId>
+        <artifactId>wildfly-quickstart-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>5</version>
+        <relativePath/>
+    </parent>
+
+    <artifactId>microprofile-lra</artifactId>
+    <version>31.0.0.Beta1-SNAPSHOT</version>
+    <packaging>war</packaging>
+    <name>Quickstart: microprofile-lra</name>
+
+    <properties>
+        <!-- the version for the Server -->
+        <version.server>30.0.0.Final</version.server>
+        <!-- The versions for the BOMs, Packs and Plugins -->
+        <version.bom.ee>${version.server}</version.bom.ee>
+        <version.bom.microprofile>${version.server}</version.bom.microprofile>
+        <version.pack.cloud>5.0.0.Final</version.pack.cloud>
```

Comment:
Please rename it to _<version.cloud.fp>5.0.0.Final</version.cloud.fp>_ in order to match https://github.com/wildfly/quickstart/blob/main/shared-doc/build-the-quickstart-for-openshift.adoc

Reply:
It should be instead fixed in the [build-the-quickstart-for-openshift.adoc](https://github.com/wildfly/quickstart/blob/main/shared-doc/build-the-quickstart-for-openshift.adoc) , this is the common name used since the enhancements

Reply:
Yes, that element is not needed since we do pluginManagement of the version. I guess that we need pluginmanagement too... Either that or get ridden of the create from scratch part, which imho provides little value, specially considering soon we will have a guide for developing apps from scratch.